start.js

```diff
@@ -0,0 +1,55 @@
+const http = require('http');
+const soldiers = require('./routes/soldiers.js');
+const duties = require('./routes/duties.js');
+const justiceBoard = require('./routes/justiceBoard.js');
+
+function runServer() {
+  let server = http.createServer(function (req, res) {
+    let body = [];
+    req.on('data', (chunk) => {
+      body.push(chunk);
+    }).on('end', () => {
+      body = Buffer.concat(body).toString();
+      if (body !== "") {
+        body = JSON.parse(body);
+      }
+      const parsedUrl = (req.url.split("/"))[1];
+      if ("soldiers" === parsedUrl ||
+        (parsedUrl.length > 14 && "soldiers?name=" === parsedUrl.substr(0, 14))) {
+        soldiers.handleSoldierReq(req, body, function (err, result) {
+          if (err) {
+            res.statusCode = 404;
+            res.end(err);
+          } else {
+            res.end(result);
+          }
+        });
+      } else if ("duties" === parsedUrl ||
+      (parsedUrl.length > 12 && "duties?name=" === parsedUrl.substr(0, 12))) {
+        duties.handleDutyReq(req, body, function (err, result) {
+          if (err) {
+            res.statusCode = 404;
+            res.end(err);
+          } else {
+            res.end(result);
+          }
+        });
+      } else if (/justiceBoard/.test(parsedUrl)) {
```

Comment:
make the justiceBoard routing more like the others.
Why?
If later on you will have another `if else <~units~>` statement that is similiar to the `soldiers` route: If a unit's name is `justiceBoard` - you will route the user to the wrong route!
Funny but true